src/kvstore/RocksEngineConfig.cpp

```diff
@@ -60,6 +60,9 @@ DEFINE_string(rocksdb_compression_per_level, "", "Specify per level compression
                                                  "e.g. \"no:no:lz4:lz4::zstd\" === "
                                                  "\"no:no:lz4:lz4:lz4:snappy:zstd:snappy\"");
 
+DEFINE_bool(rocksdb_statistics_enable, false, "Whether to export rocksdb's statistics");
```

Comment:
It would be better to name this flag to conform to others, like `--enable_partitioned_index_filter`